AndriiDryha/HW10/PRACTICAL TASKS [LESSON 10]

```diff
@@ -0,0 +1,101 @@
+# # Task 1
+
+# class Polygon:
+
+#     def __init__(self, no_of_sides):
+#         self.n = no_of_sides
+#         self.sides = [0 for i in range(no_of_sides)]
+
+#     def inputSides(self):
+#         self.sides = [float(input(f"Enter side {str(i+1)}:")) 
+#                                              for i in range(self.n)]
+
+#     def dispSides(self):
+#         for i in range(self.n):
+#             print(f"Side {i+1} is {self.sides[i]}")    
+
+# class Rectangle(Polygon):
+
+#     def __init__(self):
+#         # Polygon.__init__(self,2) 
+#         super().__init__(2) 
+
+#     def findArea(self):
```

Comment:
it is worth adding a check for the existence of a triangle
a, b = self.sides
if a > 0 and b > 0 and c > 0:

   if (a + b) > c and (a + c) > b and (c + b) > a:
       area = a * b
       print(area)
   else:
       print("This triangle doesn't exist")
else:
   print("The side length can not be negative or equal to 0!")